### DISCUSSION TITLE
When will CoPilot support IntelliJ IDEA?

### DISCUSSION THREAD

**Discussion by luoxiaofeng1029** [community/community#58264] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am very honored to receive the testing qualification, but I write Java and do not use Visual Studio Code. I usually use IntelliJ IDEA. When will IntelliJ IDEA be supported?

**Comment by danielalyoshin** [comment, marked as answer]
Copilot is already [available as a plugin](https://plugins.jetbrains.com/plugin/17718-github-copilot) for the JetBrains IDEs.

Compatibility is as follows:
> Compatible with IntelliJ IDEA (Ultimate, Community, Educational), Android Studio, AppCode, Aqua, CLion, Code With Me Guest, DataGrip, DataSpell, GoLand, JetBrains Client, MPS, PhpStorm, PyCharm (Professional, Community, Educational), Rider, RubyMine, WebStorm

Hope this helps!

**Reply by luoxiaofeng1029** [reply]
Sorry, I made a mistake in the title. It should be "copilot-x".
